server/routes/messagesRoutes.js

```diff
@@ -0,0 +1,131 @@
+const express = require('express');
+const jwt = require('jsonwebtoken');
+const mongoose = require('mongoose');
+const router = express.Router();
+const Message = require('../models/messageModel');
+const Conversation = require('../models/conversationModel');
+const verifyToken = require("../utils/verifyToken")
+
+let jwtUser = null;
```

Comment:
It's not good practice to make this a global variable. Your RESTful API should be as stateless as possible (I'm okay with the onlineUsers variable, because we decided to make that simple and not use something like Redis)